### ISSUE TITLE
Link on page "LINQ overview - .NET" goes to "no longer regularly updated" page which is confusing

### ISSUE THREAD

**Issue by studyingegret** [dotnet/docs#34480]
> Furthermore, if there isn't a library that does what you need, you can also [write your own LINQ Provider](https://learn.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2012/bb546158(v=vs.110))!

The link links to a "no longer regularly updated" page. That page says "The code examples in this topic are provided in C# and Visual Basic", but the examples are all in Visual Basic at first appearance, until the "VB" dropdown menu in the top-right corner is changed. It's confusing.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 89e7bc95-4062-e6df-7475-21973ed1becc
* Version Independent ID: 571fd6fd-95b6-6fc9-d4df-9b44da6a01b4
* Content: [LINQ overview - .NET](https://learn.microsoft.com/en-us/dotnet/standard/linq/)
* Content Source: [docs/standard/linq/index.md](https://github.com/dotnet/docs/blob/main/docs/standard/linq/index.md)
* Product: **dotnet-fundamentals**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

**Comment by gewarren** [comment]
@studyingegret I agree the statement about the code snippets in C# and Visual Basic is a little confusing, but since we are no longer updating that documentation, it doesn't meet our current bar. 

Regarding the link off to the learn.microsoft.com/previous-versions site, we have lots of those links throughout our documentation. I'm not sure there's anything to be done about it. Tagging @BillWagner for this specific instance.

**Comment by studyingegret** [comment]
I'm mainly worried that the content would be out of date :)

**Comment by studyingegret** [comment]
Hello, is this issue going to be fixed? I can still see the problem on that page.
